packages/external-db-firestore/lib/sql_filter_transformer.js

```diff
@@ -49,9 +54,13 @@ class FilterParser {
     isSingleFieldOperator(operator) {
         return [ne, lt, lte, gt, gte, include, eq, string_begins, string_ends].includes(operator)
     }
+
+    isStringBeginsOperator(operator) {
+        return operator === string_begins
+    }
```

Comment:
I think you can check this condition in the if statement.

Reply:
I prefer it to be clear what we are actually checking